config/deploy/aws.rb

```diff
@@ -5,7 +5,7 @@
 # property set.  Don't declare `role :all`, it's a meta role.
 set :stage, :aws
 set :rails_env, 'production'
-server 'ucsb.curationexperts.com', user: 'deploy', roles: [:web, :app, :db, :resque_pool]
+server 'ucsb.curationexperts.com', user: 'adrl', roles: [:web, :app, :db, :resque_pool]
```

Comment:
I just changed them all for consistency's sake; if you're using 'deploy' as the user I'll revert that one.